config.py

```diff
@@ -81,3 +89,14 @@ def update_config_path(
         config.update(user_file_config)
     else:
         raise IOError(f"Config file {config_file_path} does not exist.")
+
+
+def get_share_with(google_driver_settings) -> typing.List[str]:
```

Comment:
is not this more like config validation to be used in check above? Or maybe split between check and getter in driver?